docs/conf/prague/2021/news/cfp-open.rst

```diff
@@ -0,0 +1,50 @@
+:template: {{year}}/generic.html
+
+.. post:: April 19, 2021
+   :tags: prague-2021, website, cfp, tickets
+
+Announcing Prague Call for Proposals and Full Website
+=====================================================
+
+Today we are announcing our `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`_ and `full website <https://www.writethedocs.org/conf/prague/{{year}}/>`_.
+
+Due to the continuing global pandemic, we are keeping the Prague conference online. The conference will be held on **October 3-5, 2021**.
+This means that we're only leaving our Call for Proposals open for one month. We know that it hasn't been an easy year. Nevertheless, we'd love to see you on our online stage in October, to share your ideas with our inspiring community of documentarians.
+
+If there is something you’d really like to see a talk about this year, submit a proposal for it or refer someone else who would be good!
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, editor, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We're hoping that folks who would not be able to travel to Prague will be able to join us this time round, both as attendees and as speakers! If you've always wanted to talk about documentation but not been able to travel for whatever reason, now is the time.
+
+You can read our full `Call for Proposals <https://www.writethedocs.org/conf/prague/{{year}}/cfp/>`__ on the website.
+The Call for Proposals will be open until **Midnight {{tz}} on {{cfp.ends}}**.
+
+Tickets
+-------
+
+Tickets will go on sale during **May 2021**. In the meantime, you can already take a look at the prices for the online conference on our `Tickets <https://www.writethedocs.org/conf/prague/{{year}}/tickets/>`_ page.
+
+If you previously purchased a ticket to our Portland conference and chose to convert your ticket to a virtual conference pass, you'll receive an email from us in the next weeks to confirm your registration to the Prague virtual conference.
+
+Opportunity Grants
+------------------
+
+Our grant program offers tickets to people who would otherwise not be able to attend.
+We're excited to be able to offer free tickets to a larger number of people this year, since there are now minimal other costs to attend the event.
+
+We will open the grant application at the same time that we open ticket sales, so stay tuned for an update! 
+
```

Comment:
Would have loved a sponsorship callout here, since we already have sponsors, but alas.